source/faq.txt

```diff
@@ -61,6 +61,21 @@ tasks with the ``MongoClientSettings`` class:
 For more information on the ``MongoClientSettings`` class, see the 
 :java-docs:`API Documentation for MongoClientSettings <apidocs/mongodb-driver-core/com/mongodb/MongoClientSettings.html>`.
 
+How do I fix: "javax.net.ssl.SSLHandshakeException: extension (5) should not be presented in certificate_request"?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This is a known error that can occur when using the TLS 1.3 protocol with
+specific versions of JDK. Before connecting to your MongoDB instance or
+cluster, you need to update your JDK to one of the following patch
+versions or newer:
+
+- JDK 11.0.7
+- JDK 13.0.3
+- JDK 14.0.2
+
+For more information on this issue, see the `JSSE Reference Guide
+<https://docs.oracle.com/javase/8/docs/technotes/guides/security/jsse/JSSERefGuide.html#enabling-tls-1.3>`__.
```

Comment:
Does this link provide any info on the issue? I only see how to enable it and that it's not directly compatible with previous versions of TLS.
As discussed, let's refer them to information that shows the reported problem. How about this one?
https://bugs.openjdk.java.net/browse/JDK-8236039